sdks/python/apache_beam/typehints/batch.py

```diff
@@ -35,6 +35,7 @@
 from typing import TypeVar
 
 import numpy as np
+import torch
```

Comment:
Could you make this a separate module, following the pattern I just used for pandas: https://github.com/apache/beam/blob/master/sdks/python/apache_beam/typehints/pandas_type_compatibility.py

That way it can be separable from everything else here if `torch` is available.

Reply:
One issue with that is I haven't found a good way to re-use the BatchConverterTest logic, for now it's just duplicated in `pandas_type_compatibility_test`.

Reply:
Done.